src/System.Runtime.WindowsRuntime/src/System/Resources/WindowsRuntimeResourceManager.cs

```diff
@@ -231,7 +231,20 @@ private static void GlobalResourceContextChanged(object sender, IMapChangedEvent
             Debug.Assert(s_globalResourceContextFallBackList != null);
             Debug.Assert(s_globalResourceContext != null);
 
-            List<String> languages = new List<string>(s_globalResourceContext.Languages);
+            IReadOnlyList<string> langs;
+
+            try
+            {
+                langs = s_globalResourceContext.Languages;
+            }
+            catch (ArgumentException)
+            {
+                // Sometimes Windows Runtime fails and we get Argument Exception which can fail fast the whole app
+                // to avoid that we ignore the exception.
```

Comment:
Not exactly but this is most likely because the test is setting the current cultures a lot (kind of stress way) which cause us calling the WinRT and then causing WinRT call us back again. Every time we call WinRT or WinRT call us back we access ResourceContext.Languages. so I think in one of the callback we are getting could be ResourceContext.Languages not in stable state. causing such failure.